README.md

```diff
@@ -4,6 +4,55 @@ systems, including:
  - [mantle](https://github.com/coreos/mantle)
  - [rpm-ostree](https://github.com/projectatomic/rpm-ostree/)
  - [libvirt](https://github.com/libvirt/libvirt)
+ 
+Getting started - prerequisites
+---
+
+You can use `podman` or `docker`. These examples use `podman`. Note the
+container must be privileged, as the build process uses container functionality
+itself (i.e. we're doing [recursive containers(https://github.com/projectatomic/bubblewrap/issues/284)]).
```

Comment:
nit: formatting typo, should be:  `[recursive containers](https://github.com/projectatomic/bubblewrap/issues/284)`